src/current/_includes/v23.1/essential-metrics.md

```diff
@@ -1,54 +1,54 @@
-These essential CockroachDB metrics enable you to build custom dashboards with the following tools: 
+These essential CockroachDB metrics enable you to build custom dashboards with the following tools:
 {% if include.deployment == 'self-hosted' %}
 * [Grafana]({% link {{ page.version.version }}/monitor-cockroachdb-with-prometheus.md %}#step-5-visualize-metrics-in-grafana)
 * [Datadog Integration]({% link {{ page.version.version }}/datadog.md %}) - The [**Datadog Integration Metric Name**](https://docs.datadoghq.com/integrations/cockroachdb/?tab=host#metrics) column lists the corresponding Datadog metric which requires the `cockroachdb.` prefix.
-{% elsif include.deployment == 'dedicated' %}
-* [Datadog integration]({% link cockroachcloud/tools-page.md %}#monitor-cockroachdb-dedicated-with-datadog) - The [**Datadog Integration Metric Name**](https://docs.datadoghq.com/integrations/cockroachdb_dedicated/#metrics) column lists the corresponding Datadog metric which requires the `crdb_dedicated.` prefix.
+{% elsif include.deployment == 'advanced' %}
+* [Datadog integration]({% link cockroachcloud/tools-page.md %}#monitor-cockroachdb-cloud-with-datadog) - The [**Datadog Integration Metric Name**](https://docs.datadoghq.com/integrations/cockroachdb_dedicated/#metrics) column lists the corresponding Datadog metric which requires the `crdb_cloud.` prefix.
```

Comment:
is this change to the prefix from `crdb_dedicated` to `crdb_cloud` something that exists now or planned work?  I assume there are code changes required .  Is there an engineering issue / PR about this you could share?